packages/notifi-react-card/lib/components/NotifiEmailInput.tsx

```diff
@@ -16,16 +15,23 @@ export type NotifiEmailInputProps = Readonly<{
     label: string;
   }>;
   disabled: boolean;
+  intercomEmailInputStyle?: string;
+  email: string;
+  setEmail: (email: string) => void;
+  emailErrorMessage: string;
+  setEmailErrorMessage: (emailErrorMessage: string) => void;
```

Comment:
Is it not possible to use SubscriptionContext for Intercom?

Let's talk about architecture before we spend too much energy

Reply:
I was also thinking of using SubscriptionContext for Intercom, not sure if our tenant would use subscription edit view and intercom view on the same webpage(need clarify on the use case for intercom), if so, it's impossible to use SubscriptionContext with edit view and intercom at the same time. 
Yeah, let's talk about tomorrow in the meeting.